packages/import/tests/schema/import-schema.spec.ts

```diff
@@ -95,7 +95,7 @@ describe('importSchema', () => {
   });
 
   test('parseSDL: non-import comment', () => {
-    expect(parseSDL(`#importent: comment`)).toEqual([]);
+    expect(parseSDL(`#important: comment`)).toEqual([]);
```

Comment:
this is puzzling. I think it's just trying to prove that while it might want to process `#import`, it wouldn't parse `#important`, in which case this change is totally fine